services/src/test-library/src/main/java/org/openkilda/testing/config/DefaultServiceConfig.java

```diff
@@ -81,6 +81,14 @@ public RestTemplate labApiRestTemplate(@Value("${lab-api.endpoint}") String endp
         return buildLoggingRestTemplate(endpoint);
     }
 
+    @Bean(name = "grpcRestTemplate")
+    public RestTemplate grpcRestTemplate(
+            @Value("${grpc.endpoint}") String endpoint,
+            @Value("${grpc.username}") String username,
+            @Value("${grpc.password}") String password) {
```

Comment:
these values should be added to `kilda.properties.example` (and they should be valid for a basic `make up-test-mode` deployment)

Reply:
the 'kilda.properties.example' file has been updated